webcomponents/bundle/src/main/bundle/toolnames.properties

```diff
@@ -0,0 +1,6 @@
+assignments=Assignments
+calendar=Calendar
+grades=Grades
+tasks=Tasks
+announcements=Announcements
+forums=Forums
```

Comment:
Is it best practice to redefine these here or pull them from the other property file where the tool itself is defined?

Reply:
Not sure about that. I'd have to pull in multiple i18n bundles in the widget picker just to get the tool names. One to keep an eye on, probably. I may add a TODO on that piece.